static/app/gettingStartedDocs/javascript/remix.tsx

```diff
@@ -136,7 +136,7 @@ const replayOnboarding: OnboardingConfig = {
         {
           code: [
             {
-              label: 'JavaScript',
+              label: 'entry.client.tsx',
```

Comment:
Unrelated to feedback, but since we have similar note, I have also changed the name of the file/label to point to the client file